fabtests/benchmarks/rdm_cntr_pingpong.c

```diff
@@ -104,6 +104,8 @@ int main(int argc, char **argv)
 	hints->caps = FI_MSG;
 	hints->domain_attr->mr_mode = opts.mr_mode;
 	hints->domain_attr->threading = FI_THREAD_DOMAIN;
+	hints->domain_attr->data_progress = FI_PROGRESS_MANUAL;
```

Comment:
Do we want to force all these tests to manual progress? What if we want to test auto progress? I think it makes more sense to leave data progress alone.

Reply:
+1, I talked with Seth and we agree this should be an opt-in feature and specified as a new argument

Reply:
made it as an opt in feature.